### ISSUE TITLE
Translations does not work in AppImage

### ISSUE THREAD

**Issue by fvacek** [Quick-Event/quickbox#939]
```
2023-08-12T22:23:52[main.cpp:114]|I| Loading translations for: cs_CZ
2023-08-12T22:23:52[main.cpp:129]|I| Installing translator file: libqfcore  ...  ERROR
2023-08-12T22:23:52[main.cpp:129]|I| Installing translator file: libqfqmlwidgets  ...  ERROR
2023-08-12T22:23:52[main.cpp:129]|I| Installing translator file: libquickeventcore  ...  ERROR
2023-08-12T22:23:52[main.cpp:129]|I| Installing translator file: libquickeventgui  ...  ERROR
2023-08-12T22:23:52[main.cpp:129]|I| Installing translator file: libsiut  ...  ERROR
2023-08-12T22:23:52[main.cpp:129]|I| Installing translator file: quickevent  ...  ERROR
2023-08-12T22:23:52[main.cpp:139]|I| Installing translator file: qt ...  ERROR
```

**Comment by paukert** [comment]
Still not working, it looks like that there is `/` missing in path when using AppImage (with `/` after bin it is a valid path).

```
2023-08-27T11:53:14[main.cpp:115]|I| Loading translations for: cs_CZ search dir: /tmp/.mount_QuickEOUrft6/usr/bintranslations
2023-08-27T11:53:14[main.cpp:130]|I| Installing translator file: libqfcore  ...  ERROR
2023-08-27T11:53:14[main.cpp:130]|I| Installing translator file: libqfqmlwidgets  ...  ERROR
2023-08-27T11:53:14[main.cpp:130]|I| Installing translator file: libquickeventcore  ...  ERROR
2023-08-27T11:53:14[main.cpp:130]|I| Installing translator file: libquickeventgui  ...  ERROR
2023-08-27T11:53:14[main.cpp:130]|I| Installing translator file: libsiut  ...  ERROR
2023-08-27T11:53:14[main.cpp:130]|I| Installing translator file: quickevent  ...  ERROR
2023-08-27T11:53:14[main.cpp:140]|I| Installing translator file: qt ...  ERROR
2023-08-27T11:53:14[coreplugin.cpp:56]|I| Reports dir set to: /tmp/.mount_QuickEThYKHm/usr/bin/reports
```

**Comment by paukert** [comment]
Fixed in 3.0.5
